homeassistant/components/nordpool/strings.json

```diff
@@ -0,0 +1,56 @@
+{
+  "config": {
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "no_data": "Api connected but response was empty"
```

Comment:
```suggestion
      "no_data": "API connected but the response was empty"
```

Reply:
I'll await further code reviews and update this at that time.